config/software/gecode.rb

```diff
@@ -25,6 +25,11 @@
   source md5: "b4191d8cfafa18bd9b78594544be2a04"
 end
 
+# Major version, have not tried yet
+version "4.4.0" do
```

Comment:
Yep! I added major versions in there when they existed, but didn't switch to them.